content/en/docs/reference/kubectl/introduction.md

```diff
@@ -1,49 +1,49 @@
 ---
-title: "Introduction To Kubectl"
+title: "Introduction to kubectl"
 content_type: concept
 weight: 1
 ---
 
-Kubectl is the Kubernetes cli version of a swiss army knife, and can do many things.
+kubectl is the Kubernetes cli version of a swiss army knife, and can do many things.
 
-While this Book is focused on using Kubectl to declaratively manage Applications in Kubernetes, it
-also covers other Kubectl functions.
+While this Book is focused on using kubectl to declaratively manage applications in Kubernetes, it
+also covers other kubectl functions.
```

Comment:
This page was added as a direct [migration](https://github.com/kubernetes/website/pull/45607/files) from kubectl guide [(here)](https://kubectl.docs.kubernetes.io/guides/introduction/kubectl/). However, some of the text here seems out of place. It would greatly improve the page if we drop this initial text of referring book.

```suggestion
```